.nvmrc

```diff
@@ -0,0 +1 @@
+10.15.3
```

Comment:
I think we want to set this to `lts/dubnium`, so that we get the latest point releases. See [long-term support](https://github.com/nvm-sh/nvm#long-term-support) in the NVM docs.

Reply:
I was going to ask what you wanted to lock it to. I *think* you might want to lock it so you're not surprised when CI suddenly is using a new version 😄 but easy to see if it works